src/app/components/SearchResults/ResultsList.jsx

```diff
@@ -68,7 +68,15 @@ class ResultsList extends React.Component {
 
     return results.map((result, index) => {
       const showRequestButton = shouldShowRequest ? (
-        <a class="link" onClick={() => this.openForm(result, result.editions[0])}>Request Digitization</a>
+        <a
+          role="link"
+          tabIndex="0"
+          className="link"
+          onKeyDown={() => this.openForm(result, result.editions[0])}
```

Comment:
Should this be tied explicitly to the enter key? My event knowledge is a bit rusty but I think this would trigger the form on any key press?

Reply:
I was supressing a warning without thinking about it -- you're right, we should submit on enter